src/base/TileInternal.d.ts

```diff
@@ -0,0 +1,35 @@
+import { Tile } from './Tile';
+
+/**
+ * Internal state used/set by the package.
+ */
+
+export interface TileInternal extends Tile {
```

Comment:
We can keep this for now -- might make a good reference? If it looks like it won't be useful we can remove it later.